src/CodeStyle/VisualBasic/Tests/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.UnitTests.vbproj

```diff
@@ -5,6 +5,8 @@
     <OutputType>Library</OutputType>
     <TargetFramework>net472</TargetFramework>
     <RootNamespace></RootNamespace>
+    <DefineConstants>$(DefineConstants),CODE_STYLE</DefineConstants>
+    <IsShipping>false</IsShipping>
```

Comment:
><IsShipping>false</IsShipping> [](start = 4, length = 30)

Setting IsShipping shouldn't be necessary here.

Reply:
Yep, reverted.